docs/examples/index.rst

```diff
@@ -9,6 +9,7 @@ Tutorials
    detection_pipeline.ipynb
    augmentation_gallery.ipynb
    warp.ipynb
+   3D.ipynb
```

Comment:
maybe rename this file to `warp_3d` as we might want to have other 3d notebooks later